diff --git a/arch/arm/mach-rockchip/rk3588/Kconfig b/arch/arm/mach-rockchip/rk3588/Kconfig
index 5a433ca046c..6f76dca6a3e 100644
--- a/arch/arm/mach-rockchip/rk3588/Kconfig
+++ b/arch/arm/mach-rockchip/rk3588/Kconfig
@@ -37,6 +37,12 @@ config TARGET_EVB_RK3588
 	help
 	  RK3588 EVB is a evaluation board for Rockchp RK3588.
 
+config TARGET_G98_RK3588
+	bool "BDY G98 RK3588"
+	help
+	  RK3588 router board with two RTL8125B NICs, two YT9215
+	  switches, two PCIe 3.0 x2 M.2 slots, SATA and SPI NOR.
+
 config TARGET_CM3588_NAS_RK3588
 	bool "FriendlyElec CM3588 NAS"
 	help
@@ -458,6 +464,7 @@ source "board/friendlyelec/nanopc-t6-rk3588/Kconfig"
 source "board/friendlyelec/nanopi-r6c-rk3588s/Kconfig"
 source "board/friendlyelec/nanopi-r6s-rk3588s/Kconfig"
 source "board/gameforce/ace-rk3588s/Kconfig"
+source "board/g98/g98-rk3588/Kconfig"
 source "board/hardkernel/odroid_m2/Kconfig"
 source "board/indiedroid/nova/Kconfig"
 source "board/khadas/khadas-edge2-rk3588s/Kconfig"
